app/src/components/widgets/singleLevelWidgets/tables/components/testsTableWidget/testsTable/testsTableRow/testsTableRow.jsx

```diff
@@ -76,7 +107,23 @@ export class TestsTableRow extends Component {
           <span>{name}</span>
         </NavLink>
         {Matrix && count && (
-          <div className={cx('col', 'col-count')}>
+          <div
+            className={cx('col', 'col-count')}
+            title={
+              widgetType
+                ? formatMessage(titleMessages[widgetType], {
```

Comment:
It's not clear that widgetType will affect tooltip on matrix. Also component could be broken in case of no corresponding messages for specified widget type. What do you think about lifting up tooltip logic and pass callback `getMaxtrixTooltip` to the common components?